test/parallel/test-os.js

```diff
@@ -84,7 +84,7 @@ assert.ok(hostname.length > 0);
 // IBMi process priority is different.
 if (!common.isIBMi) {
   const { PRIORITY_BELOW_NORMAL, PRIORITY_LOW } = os.constants.priority;
-  const LOWER_PRIORITY = os.getPriority() > PRIORITY_BELOW_NORMAL ? PRIORITY_BELOW_NORMAL : PRIORITY_LOW;
+  const LOWER_PRIORITY = os.getPriority() < PRIORITY_BELOW_NORMAL ? PRIORITY_BELOW_NORMAL : PRIORITY_LOW;
   os.setPriority(LOWER_PRIORITY);
```

Comment:
Should we rename the variable?
```suggestion
  const HIGHER_PRIORITY = os.getPriority() < PRIORITY_BELOW_NORMAL ? PRIORITY_BELOW_NORMAL : PRIORITY_LOW;
  os.setPriority(HIGHER_PRIORITY);
```

Reply:
Nevermind, I read the description 😅 We should maybe add a comment then.
```suggestion
  // Priority means niceness here, higher niceness <=> lower priority
  const LOWER_PRIORITY = os.getPriority() < PRIORITY_BELOW_NORMAL ? PRIORITY_BELOW_NORMAL : PRIORITY_LOW;
  os.setPriority(LOWER_PRIORITY);
```